test/test_public_bindings.py

```diff
@@ -261,7 +261,7 @@ def test_no_new_bindings(self):
             "set_num_threads",
             "unify_type_list",
             "vitals_enabled",
-
+            "VULKAN_AUTOMATIC_GPU_TRANSFER",
```

Comment:
Do you actually want this to be public API as `torch.VULKAN_AUTOMATIC_GPU_TRANSFER` ?
Is it properly documented as well?